.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,52 @@
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Reproduction Steps
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: |
+        1. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)